web/src/main/java/org/phoenixctms/ctsms/web/jersey/resource/proband/ProbandResource.java

```diff
@@ -330,7 +310,6 @@ public Response renderInquiriesSignup(@PathParam("id") Long id, @QueryParam("dep
 		return response.build();
 	}
 
-	// @HEAD
```

Comment:
the intention was to provide details like size and mimetype about the file, without actually retrieving it.
the tru HEAD does respond headers only, but not json body with details. so an option is to write the header data explicitly.